.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Good work adding your node_modules folder to your `.gitignore` file. ✔️ However, please kindly ensure to add a `test.md` file to your `.gitignore` too as requested in the requirement below. 👇🏽
>A test.md file with the text "This file should be ignored by git".